tests/components/teslemetry/fixtures/products.json

```diff
@@ -4,7 +4,7 @@
       "id": 1234,
       "user_id": 1234,
       "vehicle_id": 1234,
-      "vin": "VINVINVIN",
+      "vin": "LRWXF7EK4KC700000",
```

Comment:
Should we add a new car instead of modifying the existing (so there's another set of tests specifically for "old" cars?

Reply:
This VIN is an old car, this also fixes the `model` in the device entry, so its cleaner to just use this VIN everywhere IMO.

Reply:
Ok. But we also have a newer car so we're testing both?

Reply:
There is added functionality for older cars, but everything else is the same. The only thing I could test with a newer VIN is that the pre2021 code block _doesnt_ run, but its just an boolean IF, so that seems to have limited value.

Reply:
Ok 👍